superset/reports/commands/execute.py

```diff
@@ -437,7 +437,7 @@ def send_error(self, name: str, message: str) -> None:
         """
         header_data = self._get_log_data()
         header_data["error_text"] = message
-        logger.info("header_data info %s", header_data)
+        logger.info("header_data info %s, taskid, %s", header_data, self._execution_id)
```

Comment:
Can we be a bit more descriptive on this log as to where it's happening? In the large scope of logs, it won't provide a whole lot of information as to what triggered this log.